docs: roadmap 2024 H1 blog (#8362)
ibis-project/ibis: docs/posts/roadmap-2024-H1/index.qmd

Jim took this on over the last few months, building up the
[IbisML](https://github.com/ibis-project/ibisml) package to a usable (but still

> early) state. We will further invest in IbisML this year to get it a

Reviewer: ```suggestion
early) state. We will further invest in IbisML this year to get it to a
```

We would love to onboard new contributors to the project.

## Other themes

> With less engineering priority, we have a few other themes for 2024.

Reviewer: ```suggestion
We have a few other themes for 2024 that are lower priority.
```

## Beyond Q1 2024

> This writeup of our roadmap is heavily biased toward Q1 of 2024. Looking beyond,

Reviewer: ```suggestion
This writeup of our roadmap is heavily focused on Q1 of 2024. Looking beyond,
```

Biased feels like the wrong word to me here.
